tests/containers/podman_bci_systemd.pm

```diff
@@ -21,7 +21,10 @@ sub run {
     my $image = get_var("CONTAINER_IMAGE_TO_TEST", "registry.suse.com/bci/bci-init:latest");
 
     record_info('Test', 'Launch a container with systemd');
-    assert_script_run("podman run -d -p 80:80 --name nginx $image");
+    assert_script_run("podman run -d -p 80:80 --health-cmd='curl http://localhost' --name nginx $image");
+
+    # Wait for the container to be up
+    script_retry("podman ps | grep nginx | grep healthy", retry => 10, delay => 15);
```

Comment:
Let's use inspect for that.  Also the `-x` option is needed in case it returns "unhealthy".

```suggestion
    script_retry("podman inspect -f '{{.State.Health.Status}}' nginx | grep -x healthy", retry => 10, delay => 15);
```

Reply:
`podman container wait --condition=healthy` should also work, btw.